src/borg/cache.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, repository, key, manifest, path=None, sync=True, do_files=Fal
         self.key = key
         self.manifest = manifest
         self.path = path or os.path.join(get_cache_dir(), repository.id_str)
+        if os.environ.get('BORG_UNIQ_HOSTNAME'):
+            self.uniq_hostname = True
+        else:
+            self.uniq_hostname = False
```

Comment:
just for your python powers: equivalent is this one-liner:

x = bool(os.environment.get(...))